src/components/fields/FilterOperation.js

```diff
@@ -132,10 +134,40 @@ UnconnectedFilterOperation.propTypes = {
   updatePlot: PropTypes.func,
   ...Field.propTypes,
 };
+UnconnectedFilterOperation.contextTypes = {
+  localize: PropTypes.func,
+};
 
 class UnconnectedFilterValue extends Component {
   constructor(props, context) {
     super(props, context);
+    const {localize: _} = context;
+
+    // Replicated so that strings can be statically analyzed for localization
```

Comment:
why is replication needed?

Reply:
My guess was that there's a script that does static analysis of the code and adds strings to the translation files, so this thing needs to live inside a component. Tried putting it in one and accessing from another - like UnconnectedFilterValue.operation - no luck. What's the way?

Reply:
@VeraZab ?

Reply:
you could make it a function? and pass in localize as a prop?